content/2.developers/3.guides/6.multisig/2.navigate-the-multiSig-interface.md

```diff
@@ -0,0 +1,102 @@
+
+Let's get an overview of the other proposal types on this window:
+- **Treasury Spend**: The treasury spend proposal allows for sending native tokens from the Multisig account to another account.
+- **Voting Configuration**: This proposal type allows you to update the voting configuration for the multisig.
+- **Governance**: This lists the general governance proposals that can be created on chain.
+- **Execute Contract**: Allows for executing a message on a smart contract. You will need to know the structure of such a message as the proposal content will have a JSON message that highlights the entry point to be executed and the actual message that should can be submitted. You will enter the smart contract to which this message should be submitted.
+
+As an example let's execute a **Treasury Spend** proposal. Make sure to send some **ARCH** or **CONST** tokens to the Multisig account. You can find the multisig account address by clicking the drop down menu in the left navigation bar and clicking on the address to copy it.
+
+1. Click "Treasuty Spend"
+2. Enter a proposal name
+3. Enter a description of the proposal
+4. Enter the token amount and address you would like to transfer funds to
+5. Click the "Spend" button
+
+![](/images/docs/multisig-ui/multisig-treasury-spend.png)
+
+This proposal will now be added to the Propsals page. To vote on the proposal click on the proposal and a window will be shown with the details of the proposal. Click on the "Vote" button and select the vote "Yes", "No" or "Abstain".
+
+![](/images/docs/multisig-ui/multisig-treasury-spend-proposal.png)
+
+Once the proposal is approved it will be ready to be executed. It should show up under the "**To Execute**" tab. Click on the proposal and click the "**Execute Proposal**" button to complete the transaction.
+
+![](/images/docs/multisig-ui/multisig-exeute.png)
+
+Once the transaction is success the proposal should now show up under the "**Executed**" tab.
+
+![](/images/docs/multisig-ui/multisig-executed.png)
+
+
+## Staking
+
+Staking is a popular transaction to execute on chain and the reason it is added here as it’s own page. So any action executed here will actually create a staking related proposal. The main actions that can be executed on the Staking page include:
+
+- Claiming staking rewards
+- Stake tokens to a validator
+- Unstake tokens from a validator
+- Redelegate tokens from one validator to another validator
+
+
+## Governance
+
+The governance page lists all governance proposals created on the Archway blockchain. It allows the multisig account to vote on these proposals. If you see a governance proposal that’s still active you can click on the governance proposal and select a vote (Yes, No, Abstain, No with veto) which will generate a proposal for members to vote on. If the proposal is approved the vote made on the governance proposal will be submitted on chain.
```

Comment:
Iit would be nice to make it more notorious the difference between a 1. "proposal" and a 2. "governal proposal", maybe for the first of the 2, instead of just "proposal" we can call it a "multisig proposal" instead?

Reply:
That came to mind when I was writing the governance component and I tried to always say "Governance Proposal" when it's governance base but it might still be a bit confusing. Will have to look back at how best to word it.